miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebViewClient.kt

```diff
@@ -17,11 +17,17 @@ internal class MiniAppWebViewClient(
     override fun shouldInterceptRequest(
         view: WebView,
         request: WebResourceRequest
-    ): WebResourceResponse? {
+    ): WebResourceResponse? = loader.shouldInterceptRequest(request.url)
+
+    override fun onReceivedError(
+        view: WebView,
+        request: WebResourceRequest,
+        error: WebResourceError
+    ) {
         if (request.url != null && request.url.toString().startsWith(customScheme)) {
-            val interceptUri = request.url.toString().replace(customScheme, customDomain).toUri()
-            return loader.shouldInterceptRequest(interceptUri)
+            view.loadUrl(request.url.toString().replace(customScheme, customDomain))
```

Comment:
I was thinking, in `RealMiniAppDisplay` should we also change it to directly load the `https` url instead of the custom scheme? That way we don't have to hit an error on the initial load. 

I know that then it seems kind of pointless to have the `mscheme` at all, but the main purpose of the custom scheme is for having a consistent callback URI across platforms.